test/powershell/Language/Scripting/NativeExecution/NativeCommandProcessor.Tests.ps1

```diff
@@ -128,6 +128,18 @@ Describe "Native Command Processor" -tags "Feature" {
             $ps.Dispose()
         }
     }
+
+    It "OutputEncoding should be used" -Skip:(!$IsWindows -or !(Get-Command sfc.exe)) {
+
+        $originalOutputEncoding = [Console]::OutputEncoding
+        try {
+            [Console]::OutputEncoding = [System.Text.Encoding]::Unicode
+            sfc | Out-String | Should -Not -Match "`0"
```

Comment:
sfc.exe requires elevated rights. Also I run the exe in latest build and I don't see problem.

Reply:
It does, but the error it generates running in non-privileged mode is still in Unicode so will still work for this test. 🙂